src/callbacks.ts

```diff
@@ -61,7 +43,7 @@ export abstract class Callback {
   // TODO(michaelterry): This type is a best guess.
   validationData: Tensor|Tensor[] = null;
   /** Instance of `keras.models.Model`. Reference of the model being trained. */
-  model: Model = null;
+  model: Container = null;
```

Comment:
All the Model->Container changes here have me a tiny bit worried as we're changing the API contract.